src/plugins/intel_cpu/tests/functional/custom/behavior/import_model_fuzzy.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <sstream>
+#include <fstream>
+
+#include "openvino/runtime/core.hpp"
+#include "openvino/op/parameter.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/matmul.hpp"
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "openvino/pass/serialize.hpp"
+
+static std::shared_ptr<ov::Model> getModel() {
+    auto type = ov::element::f32;
+    auto input = std::make_shared<ov::op::v0::Parameter>(type, ov::Shape{16, 8});
+    auto tensor = ov::test::utils::create_and_fill_tensor(type, ov::Shape{8, 16});
+    auto weight = std::make_shared<ov::op::v0::Constant>(tensor);
+    auto matmul = std::make_shared<ov::op::v0::MatMul>(input, weight);
+    return std::make_shared<ov::Model>(matmul, ov::ParameterVector{input}, "import_model");
+}
+
+TEST(CPUFuzzyImportModel, RunWithCompiledModel) {
```

Comment:
it would be great to share such tests as non-CPU specific. E.g. move to this place https://github.com/openvinotoolkit/openvino/tree/master/src/tests/functional/plugin/shared/include/behavior/ov_plugin and parametrize with device name

@sbalandi @iefode do we have a better place for such tests?

Reply:
@ilya-lavrenov @zhangYiIntel 

We have the same tests in OP conformance per conformance IR: https://github.com/openvinotoolkit/openvino/blob/b20bc815145dc53598eb05c8ad920f16721d549b/src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/src/read_ir/read_ir.cpp#L336

Also, we can extend API conformance suite by these tests in https://github.com/openvinotoolkit/openvino/blob/b20bc815145dc53598eb05c8ad920f16721d549b/src/tests/functional/plugin/shared/src/behavior/compiled_model/import_export.cpp#L54

@zhangYiIntel Could you please extend conformance?

Reply:
Sure, let me try this.